gluecodium/src/main/resources/templates/cpp/CppEnumeration.mustache

```diff
@@ -26,4 +26,16 @@ enum class {{>cpp/CppAttributesInline}}{{resolveName}} {
 }}    {{resolveName}}{{#explicitValue}} = {{resolveName}}{{/explicitValue}}{{#if iter.hasNext}},{{/if}}
 {{/enumerators}}
 };
+
+{{#if attributes.cpp.tostring}}
+std::string_view
+to_string({{resolveName}} foo) {
+  switch (foo) {
+    {{#enumerators}}
+    case {{resolveName}}{{explicitValue}}: return "{{resolveName}}{{explicitValue}}";
```

Comment:
This will not compile. Name of the enum type is missing, `::` is missing, `explicitValue` is not needed at all. I think the correct template is something like
```
    {{#set enum=this}}{{#enumerators}}
    case {{resolveName enum}}::{{resolveName}}: return "{{resolveName enum}}::{{resolveName}}";
    {{/enumerators}}{{/set}}
```